docs/source-2.0/aws/protocols/aws-json-1_1-protocol.rst

```diff
@@ -128,4 +128,28 @@ The following example defines a service that requires the use of
 .. |protocol test link| replace:: https://github.com/awslabs/smithy/tree/main/smithy-aws-protocol-tests/model/awsJson1_1
 .. include:: aws-json.rst.template
 
+
+--------------------
+Error shape renaming
+--------------------
+
+By default, Smithy permits renaming shapes to disambiguate shape ID conflicts in
```

Comment:
I think this paragraph and the next paragraph should be combined (we don't need this many one sentence paragraphs).